src/core/gui/toolbarMenubar/ToolMenuHandler.cpp

```diff
@@ -538,6 +538,9 @@ void ToolMenuHandler::initToolItems() {
      */
     addToolItem(new ColorToolItem(listener, toolHandler, this->parent, NamedColor{}, true));
 
+    addCustomItem("ACTION_X_REFLECTION", ACTION_X_REFLECTION, "x-reflect", _("x-reflection"));
+    addCustomItem("ACTION_Y_REFLECTION", ACTION_Y_REFLECTION, "y-reflect", _("y-reflection"));
```

Comment:
Better naming for the `Customize Toolbars` window:
```suggestion
    addCustomItem("ACTION_X_REFLECTION", ACTION_X_REFLECTION, "x-reflect", _("Reflect About X-Axis"));
    addCustomItem("ACTION_Y_REFLECTION", ACTION_Y_REFLECTION, "y-reflect", _("Reflect About Y-Axis"));
```

Reply:
Done in `d68cc98`